apps/website-25/package.json

```diff
@@ -36,6 +36,7 @@
     "react-click-away-listener": "^2.3.0",
     "react-device-detect": "^2.2.3",
     "react-dom": "^18.2.0",
+    "react-icons": "^5.5.0",
```

Comment:
This is not actually a new dep - it's pulled in from ui already. But eslint needs this not to complain (semi rightfully so, because if we uninstall libraries/ui it would break which would be unexpected).